src/test/java/org/ditacommunity/xml/grammarpool/TestGrammarPoolUsingSAXParserFactory.java

```diff
@@ -0,0 +1,51 @@
+package org.ditacommunity.xml.grammarpool;
+
+import org.apache.xerces.parsers.XMLParser;
+import org.apache.xerces.xni.grammars.XMLGrammarPool;
+import org.apache.xml.resolver.tools.CatalogResolver;
+import org.junit.Test;
+import org.xml.sax.HandlerBase;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+
+import java.net.URL;
+import java.util.Date;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class TestGrammarPoolUsingSAXParserFactory {
+
+    @Test
+    public void getParser() throws Exception {
+        GrammarPoolUsingSAXParserFactory factory = (GrammarPoolUsingSAXParserFactory)GrammarPoolUsingSAXParserFactory.newInstance();
```

Comment:
Need to set up a local XML catalog so the parser can succeed without hard coding to an OT instance somewhere.